tests/wpt/meta/css/cssom-view/elementsFromPoint.html.ini

```
@@ -1,4 +1,7 @@
 [elementsFromPoint.html]
+  [SVG element at x,y]
+    expected: FAIL
```

```suggestion
    expected: [PASS, FAIL]
```

Seems like this is intermittent on the CI as this passes consistently for me locally. I'll look into it.

---

Suppressing the failure with #38612 has the benefit of allowing us to track whether or not the test is still flaky.